java/ingest/ingest-runner/src/test/java/sleeper/ingest/runner/impl/IngestCoordinatorFileWritingStrategyIT.java

```diff
@@ -84,16 +86,17 @@ public class IngestCoordinatorFileWritingStrategyIT extends LocalStackTestBase {
     private final TestIngestType ingestType = directWriteBackedByArrowWriteToLocalFile();
     private final Instant stateStoreUpdateTime = Instant.parse("2023-08-08T11:20:00Z");
     private StateStore stateStore;
+    private SketchesStore store;
```

Comment:
This field name could be confusing as there's more than one store in this context. You could easily think it means the state store. It could be sketchesStore.

Since this field always has the same value, it could also be clearer if we make this field final and set the value here.